.github/workflows/test-spras.yml

```diff
@@ -81,6 +81,8 @@ jobs:
         docker pull reedcompbio/pathlinker:latest
         docker pull reedcompbio/meo:latest
         docker pull reedcompbio/mincostflow:latest
+        docker pull erikliu24/rwwr:latest
+        docker pull erikliu24/tiedie:latest
```

Comment:
Adding a reminder to change these to `reedcompbio` after @annaritz pushes the containers to the organization account. Same goes for the steps below.

Reply:
Yes, I will make changes to this file when Anna pushes the image to `reedcompbio`. And the changes will be made also in `src/random_walk.py` and `src/tiedie.py` as the `run` functions in these two files are also pulling images from my personal account.

Reply:
I built and pushed both of these to DockerHub so you can make the test workflow changes now:
- https://hub.docker.com/repository/docker/reedcompbio/random-walk-with-restart
- https://hub.docker.com/repository/docker/reedcompbio/tiedie